### ISSUE TITLE
Encryptr Web Client

### ISSUE THREAD

**Issue by AMPBEdu** [devgeeks/Encryptr#192]
It'd be cool if there was a web browser client for devices you might use that you can't install encryptr on. Would this cause security problems? Would be willing to contribute, not sure how far my knowledge would be able to go.

**Comment by devgeeks** [comment]
Yeah, the issue is primarily one of security.

There is the deliverability problem, mostly. You can't be sure the code delivered to your browser is the code I published since it could be modified along the way.

http://tonyarcieri.com/whats-wrong-with-webcrypto

**Comment by devgeeks** [comment]
That said. It *might* be possible to distribute a version in a folder that could be run completely locally. There are no native dependancies except for the ability to copy to the clipboard, and most browsers have a way to do that from JavaScript now.

I do 90% of my development on Encryptr literally by opening index.html in a browser locally. It's not the most sophisticated shipped package, but it would run nearly anywhere that had a decent browser.

**Comment by AMPBEdu** [comment]
Oh so you'd download the html file just like any other version of the client, but it'd be able to work without any installation. That would be perfect. Especially because i'm having trouble installing encryptr on a flash drive.

**Comment by devgeeks** [comment]
Yeah. I was also thinking it would solve the problem of installing on a flash drive...

**Comment by AMPBEdu** [comment]
Any eta on this? <3

**Comment by devgeeks** [comment]
No ETA yet. I have found a nice JS clipboard library that should be one of the only things holding it back.
